feature/xhr/aggregate/index.js

```diff
@@ -92,6 +94,13 @@ function storeXhr(params, metrics, startTime, endTime, type) {
     callbackDuration: metrics.cbTime
   }
 
+  if (xhrContext.dt) {
+    event.dt = {}
```

Comment:
We could put these values directly on the event object, so that we don't have to create another object.